components/prombench/manifests/benchmark/2_loadgen.yaml

```diff
@@ -73,13 +73,11 @@ spec:
       serviceAccountName: loadgen-scaler
       containers:
       - name: prom-load-generator
-        image: gcr.io/prometheus-test-204522/prometheus-bench-loadgen:v0.1.0
+        image: docker.io/sipian/prombench-loadgen:v2.0.0
```

Comment:
seems the changes in this file shouldn't belong in this PR

Reply:
Since I have also replaced env var with args in loadgen, I think this is valid.

Reply:
I have replaced `gcr.io/prometheus-test-204522/prometheus-bench-loadgen` with `docker.io/sipian/prombench-loadgen` because `gcr.io/prometheus-test-204522/prometheus-bench-loadgen` might become inactive after the free trial ends